### PR TITLE
Use nproc to get processor count

### PR DESCRIPTION
**Pull request by brandonpaiz**

### CODE REVIEW

File: containers/pre_built_workers/java/run_worker.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright 2020 gRPC authors
```
Comment (paulosjca) [inline containers/pre_built_workers/java/run_worker.sh:2]: Since this is a new file, 2021 would be appropriate.

**Comment by brandonpaiz** [comment]
Corresponding PR https://github.com/grpc/grpc/pull/27919

**Review by paulosjca — COMMENTED** [review]
Don't you need similar changes to the examples, and perhaps also to the load test configuration templates in grpc/grpc? If you don't change the templates, this script will not be called. But then this could be done in follow-up PRs.

You should update the basic examples in a separate PR just before the next release of the controller, and update the basic templates in grpc/grpc once the release is complete.

File: containers/pre_built_workers/java/run_worker.sh (hunk 1 above)
Comment (brandonpaiz) [reply containers/pre_built_workers/java/run_worker.sh]: Updated 👍

**Review by paulosjca — COMMENTED** [review]
There should be a Dockerfile for non-prebuilt workers as well, since we are already updating the examples.

**Review by paulosjca — APPROVED** [review]

**Comment by jtattermusch** [comment]
Thanks for the fix. Please fill in the PR description to describe the context in a sentence or two.

**Pull request merged**